### DISCUSSION TITLE
About the bug of tool generate-timestamp

### DISCUSSION THREAD

**Discussion by huanmin123** [he3-app/he3-feedback#319] [Bug Report]
### Version
generate-timestamp 1.0.4

### Environment
Windows v2.0.0

### Steps to reproduce
时间选择器是坏的

**Comment by zhenweiwang1990** [comment]
你是指这个么，我这里看起来是好的，你使用的是最新版本？

<img width="1014" alt="image" src="https://github.com/he3-app/he3-feedback/assets/1728779/cfc162b1-7754-498f-b02d-4dccc06297f3">
